onnxruntime/core/graph/graph_utils.h

```diff
@@ -32,6 +32,12 @@ bool IsSupportedProvider(const Node& node,
     fed to multiple downstream operators, i.e., it can have multiple output edges. */
 bool IsSingleInSingleOutNode(const Node& node);
 
+/** Checks for nodes with >= 1 outputs, if only one of the outputs is input to downstream Operators. */
+bool IsSingleOutputUsed(const Node& node);
```

Comment:
Maybe we can keep this internal for now? As in not expose it in the header but keep it as static in the cc file.

Reply:
yarp